config/jobs/kubernetes/kops/kops-config.yaml

```diff
@@ -76,7 +76,7 @@ presubmits:
         - --aws-cluster-domain=test-cncf-aws.k8s.io
         - --check-leaked-resources=false
         - --cluster=
-        - --extract=ci/latest
+        - --extract=release/latest-1.15
```

Comment:
not sure is `release` correct, the only hit what I could find is here https://github.com/kubernetes/test-infra/blob/97a313168ba509db0f4926c65b41c171e68d0636/config/jobs/kubernetes/sig-release/release-branch-jobs/1.15.yaml#L17 (and its `ci/latest-1.15`)

Reply:
https://github.com/kubernetes/test-infra/blob/master/docs/kubernetes-versions.md says that it could work

Reply:
cc @krzyzacy do we have this written down anywhere?

Reply:
only thing I can find is https://github.com/kubernetes/test-infra/tree/master/kubetest#extract-a-build

Reply:
The logic appears to be https://github.com/kubernetes/test-infra/blob/master/kubetest/extract_k8s.go